### PR TITLE
Update runningFold and runningReduce KEEP

### PR DESCRIPTION
**Pull request by qurbonzoda**

### CODE REVIEW

**Comment by qurbonzoda** [comment]
It is easier to review the commits. `Files changes` doesn't show diff, because I have renamed the KEEP file.

File: proposals/stdlib/scan-and-scanReduce.md [hunk 1]
```diff
@@ -181,22 +105,20 @@ Open review: https://upsource.jetbrains.com/kotlin/review/KOTLIN-CR-3763
...
 regardless of using initial accumulator. In Kotlin to be consistent with existing `fold` and `reduce` functions 
 we need to name variants of `scan` with and without initial value differently.
-* `scan...` is used as a prefix for the aggregate functions to indicate that variant returns all intermediate results. 
-The `fold` variant will be called just `scan` due to its use cases being more widespread.
 
 ## Contracts
 
-* `scan` returns a list containing the `initial` argument when receiver is empty
-* `scanReduce` returns an empty list when receiver is empty
+* `runningFold` returns a list containing the `initial` argument when receiver is empty
```
Comment (ilya-g) [inline proposals/stdlib/scan-and-scanReduce.md]: `runningFold`/`scan` return

**Review by ilya-g — APPROVED** [review]
Please add another commit reintroducing the old file with a link to the new file, like here https://github.com/Kotlin/KEEP/blob/master/proposals/stdlib/durations-and-clocks.md

File: proposals/stdlib/scan-and-scanReduce.md (hunk 1 above)
Comment (qurbonzoda) [reply proposals/stdlib/scan-and-scanReduce.md]: done

**Pull request merged**